new pricing add-ons and fonts

jakekvm01 opened the pull request:
Adding new pricing add-ons.

jakekvm01 commented:
Kindly check the new pricing add-ons.

MicahZoltu commented on source/index.html:
```diff
@@ -170,6 +170,15 @@
 			background: #24CDE8;
 		}
+		
+	           h1 {
+			font-weight: 300;
+			font-size: 40px;
+			line-height: 78px;
+			text-transform: uppercase;
+			color: #404040;
+                         }  
```
Some of these lines start with spaces instead of tabs.  I believe all of the files in this project use tabs for indentation, so we should follow that consistently.  Most editors have a setting that will automate this, my guess is that the editor you are using is defaulting to spaces when you press the tab key for some reason.

MicahZoltu replied:
Note: The same problem occurs throughout this document.  Easiest way to fix it is probably to enable whitespace rendering in your editor (how you do this varies by editor) and look for spaces at the beginning of any lines.

jakekvm01 replied:
Thank you. I haven't used an editor when I edited the page before (I only used notepad) :)

MicahZoltu commented on source/index.html:
```diff
@@ -766,6 +775,99 @@ <h3 class='title'>Pricing</h3>
+					#addonsbanner > .addonsbanner {
+						grid-column: 1;
+						grid-row: 1;
+
```
```suggestion
```
Empty lines should be intentional and uniformly applied throughout the file.  The rest of this file doesn't have a trailing newline inside style blocks, so we shouldn't here either.

MicahZoltu commented on source/index.html:
```diff
@@ -766,6 +775,99 @@ <h3 class='title'>Pricing</h3>
+					padding-top: 20px;
+					padding-bottom: 20px;
+				}
+
+				
+				
```
```suggestion
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,6 +775,99 @@ <h3 class='title'>Pricing</h3>
+					padding-bottom: 50px;
+				}
+				#addons > .pricing-specs > .freebie {
+					flex-grow: 1;
+				}
+				@media screen and (max-width: 680px) {
+					#addons {
+						flex-direction: column;
+						align-items: center;
+					}
+					#addons> .pricing-specs.month {
+						grid-column: 1;
+						grid-row: 1;
+					
+					              }
+				}
+			</style>
+
+				
+			
+			<div class='pricing-specs month'>
+				<div class='amount'>$100</div>
+				<div class='per'><span>Per</span></div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Chat)</div>
+				<button id='addonchat'>Details</button>
+			</div>
+			<div class='pricing-specs month'>
+				<div class='amount'>$200</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Non-chat)</div>
+				<button id='addonnonchat'>Details</button>
+                         </div>
+			
+
+              </section>
+		
+	
+		
```
Formatting cleanup.  As mentioned above, we should try to maintain a consistent style across the document, which includes tabs for indentation, blank lines only between sections, etc.
```suggestion
			</style>
			]<div class='addonsbanner'>
				<h1> + ADD-ONS </h1>
			</div>
		</section>

		<section id='addons'>
			<style>
				#addons {
					grid-column: 1;
					grid-row: 14;
					display: flex;
					justify-content: space-around;
					align-items: stretch;
					row-gap: 50px;
					padding-top: 20px;
					padding-bottom: 50px;
				}
				#addons > .pricing-specs > .freebie {
					flex-grow: 1;
				}
				@media screen and (max-width: 680px) {
					#addons {
						flex-direction: column;
						align-items: center;
					}
					#addons> .pricing-specs.month {
						grid-column: 1;
						grid-row: 1;
					}
				}
			</style>
			<div class='pricing-specs month'>
				<div class='amount'>$100</div>
				<div class='per'><span>Per</span></div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Chat)</div>
				<button id='addonchat'>Details</button>
			</div>
			<div class='pricing-specs month'>
				<div class='amount'>$200</div>
				<div class='per'>Per</div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Non-chat)</div>
				<button id='addonnonchat'>Details</button>
			</div>
		</section>
```

MicahZoltu commented on source/index.html:
```diff
@@ -978,6 +1080,42 @@ <h3 class='title'>Pricing</h3>
+						The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Overage per resolved case will be an additional $5.
+                                        </p>
+				</div>
+			        </div>                 
```
There shouldn't be extraneous trailing whitespace on any lines either.  Many editors have a feature that automatically fixes this for you.
```suggestion
			</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -170,6 +170,15 @@
 			background: #24CDE8;
 		}
+		
+	           h1 {
+			font-weight: 300;
+			font-size: 40px;
+			line-height: 78px;
+			text-transform: uppercase;
+			color: #404040;
+                         }  
```
I generally try to avoid doing page-wide style changes like this.  In this case, there is a single element on the page that you want to style this way, so we should probably apply the style locally instead of globally.  This means a style like `#addonsbanner > .addonsbanner > h1 { ... }` in the `<style>` block for the section that has this element.  This way, only that element is targeted by this style rather than every `h1` element on the page.

MicahZoltu reviewed (changes requested):
Feedback mostly around code style/cleanup.  There is one comment about the h1 style being moved that is actually impactful (not just visual consistency in the code).

Overall looks good.  I'll take another look once the changes mentioned here are applied.

jakekvm01 commented:
Please check @MicahZoltu  if I have done it correctly. Thank you :)

MicahZoltu commented on source/index.html:
```diff
@@ -191,15 +190,15 @@
 			}
 		</style>
-		<div id='nav-intro-background' class='background'>
+		    <div id='nav-intro-background' class='background'>
```
```suggestion
		<div id='nav-intro-background' class='background'>
```

jakekvm01 replied:
I will just remove the unused fonts from the folder for now.

MicahZoltu commented on source/index.html:
```diff
@@ -191,15 +190,15 @@
 				}
 			</style>
-		</div>
+		    </div>
```
```suggestion
		</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -429,7 +428,7 @@ <h1>Supporting your <b>Ethereum application</b> so you can focus on building!</h
 			<div class='image'><svg viewBox='0 0 260 251'></svg></div>
 		</section>
-		<div class='intro-seperator'>
+		    <div class='intro-seperator'>
```
```suggestion
		<div class='intro-seperator'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -475,8 +474,8 @@ <h1>Supporting your <b>Ethereum application</b> so you can focus on building!</h
 			</a>
-		</div>
-		<div class='service-background'>
+		    </div>
+		    <div class='service-background'>
```
```suggestion
		</div>
		<div class='service-background'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -490,7 +489,7 @@ <h1>Supporting your <b>Ethereum application</b> so you can focus on building!</h
 				}
 			</style>
-		</div>
+		    </div>
```
```suggestion
		</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -545,7 +544,7 @@ <h2 class='title'>our service</h2>
 			<div class='image'><svg viewBox='0 0 602 431'></svg></div>
 		</section>
-		<div style='height:75px'></div>
+		    <div style='height:75px'></div>
```
```suggestion
		<div style='height:75px'></div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -603,8 +602,8 @@ <h2 class='title'>about us</h2>
 		</section>
-		<div style='height:35px; grid-column: 1; grid-row: 7;'></div>
-		<div style='height:35px; grid-column: 1; grid-row: 8;'></div>
+		    <div style='height:35px; grid-column: 1; grid-row: 7;'></div>
+		    <div style='height:35px; grid-column: 1; grid-row: 8;'></div>
```
```suggestion
		<div style='height:35px; grid-column: 1; grid-row: 7;'></div>
		<div style='height:35px; grid-column: 1; grid-row: 8;'></div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -692,8 +691,8 @@ <h3 class='title'>Pricing</h3>
 			</style>
-		</div>
-		<div id='pricing-seperator'>
+		    </div>
+		    <div id='pricing-seperator'>
```
```suggestion
		</div>
		<div id='pricing-seperator'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -1010,6 +1111,9 @@ <h3 class='title'>Pricing</h3>
 		document.querySelector('#month-pricing-button').addEventListener('click', () => openModal('#month-fee-modal-content'))
 		document.querySelector('#message-pricing-button').addEventListener('click', () => openModal('#message-fee-modal-content'))
 		document.querySelector('#question-pricing-button').addEventListener('click', () => openModal('#question-fee-modal-content'))
+                document.querySelector('#addonchat').addEventListener('click', () => openModal('#addonchat-content'))
+		document.querySelector('#addonnonchat').addEventListener('click', () => openModal('#addonnonchat-content'))
+		
```
```suggestion
		document.querySelector('#addonchat').addEventListener('click', () => openModal('#addonchat-content'))
		document.querySelector('#addonnonchat').addEventListener('click', () => openModal('#addonnonchat-content'))
```

MicahZoltu commented on source/index.html:
```diff
@@ -935,13 +1002,13 @@ <h3 class='title'>Pricing</h3>
 				</div>
-			</div>
-			<div id='message-fee-modal-content' class='closed'>
+			    </div>
+			    <div id='message-fee-modal-content' class='closed'>
```
```suggestion
			</div>
			<div id='message-fee-modal-content' class='closed'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -912,13 +979,13 @@ <h3 class='title'>Pricing</h3>
-		<div id='modal-backdrop'></div>
-		<div id='modal-container'>
-			<div id='month-fee-modal-content' class='closed'>
+		        <div id='modal-backdrop'></div>
+		        <div id='modal-container'>
+			    <div id='month-fee-modal-content' class='closed'>
```
```suggestion
		<div id='modal-backdrop'></div>
		<div id='modal-container'>
			<div id='month-fee-modal-content' class='closed'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -912,13 +979,13 @@ <h3 class='title'>Pricing</h3>
-					<div class='amount'>$500</div>
-					<div class='per'><span>Per</span></div>
-					<div class='unit'>Month</div>
+				<div class='amount'>$500</div>
+				<div class='per'><span>Per</span></div>
+				<div class='unit'>Month</div>
```
```suggestion
					<div class='amount'>$500</div>
					<div class='per'><span>Per</span></div>
					<div class='unit'>Month</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -935,13 +1002,13 @@ <h3 class='title'>Pricing</h3>
-					<div class='per'>Per</div>
-					<div class='unit'>Message</div>
-					<div class='freebie'>(First 20,000 free)</div>
+				<div class='amount'>$0.04</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Message</div>
+				<div class='freebie'>(First 20,000 free)</div>
```
```suggestion
					<div class='amount'>$0.04</div>
					<div class='per'>Per</div>
					<div class='unit'>Message</div>
					<div class='freebie'>(First 20,000 free)</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -977,8 +1044,42 @@ <h3 class='title'>Pricing</h3>
 				</div>
-			</div>
-		</div>
+			    </div>
```
```suggestion
			</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,20 +765,88 @@ <h3 class='title'>Pricing</h3>
 				}
 			</style>
-		</div>
+			</div>
```
```suggestion
		</div>
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/');
-	src: local('Proxima Nova'), local('ProximaNovaT-Thin'),
-		url('./fonts/?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNovaT-Thin.woff') format('woff'),
-		url('./fonts/ProximaNovaT-Thin.ttf') format('truetype');
-	font-weight: 100;
-	font-style: normal;
-}
```
Is there no thin equivalent for creato?

jakekvm01 replied:
There is. Adding it now. Forgot to include it in webfont generator.

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-		url('./fonts/ProximaNova-Light.ttf') format('truetype');
-	font-weight: 300;
-	font-style: normal;
+    font-family: 'creato';
+     src: url('./fonts/');
+    src: url('./fonts/creatodisplay-light-webfont.eot');
+    src: local('creato'), local('creatodisplay-light-webfont'),
+         url('./fonts/creatodisplay-light-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-light-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-light-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-light-webfont.ttf') format('truetype');
+    font-weight: 300;
+    font-style: normal;
```
```suggestion
	font-family: 'creato';
	src: url('./fonts/');
	src: url('./fonts/creatodisplay-light-webfont.eot');
	src: local('creato'),
		local('creatodisplay-light-webfont'),
		url('./fonts/creatodisplay-light-webfont.eot?#iefix') format('embedded-opentype'),
		url('./fonts/creatodisplay-light-webfont.woff') format('woff'),
		url('./fonts/creatodisplay-light-webfont.woff2') format('woff2'),
		url('./fonts/creatodisplay-light-webfont.ttf') format('truetype');
	font-weight: 300;
	font-style: normal;
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-		url('./fonts/ProximaNova-Regular.ttf') format('truetype');
-	font-weight: normal;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-black-webfont.eot');
+    src: local('creato'), local('creatodisplay-black-webfont'),
+         url('./fonts/creatodisplay-black-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-black-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-black-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-black-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
```
```suggestion
	font-family: 'creato';
	src: url('./fonts/creatodisplay-black-webfont.eot');
	src: local('creato'),
		local('creatodisplay-black-webfont'),
		url('./fonts/creatodisplay-black-webfont.eot?#iefix') format('embedded-opentype'),
		url('./fonts/creatodisplay-black-webfont.woff') format('woff'),
		url('./fonts/creatodisplay-black-webfont.woff2') format('woff2'),
		url('./fonts/creatodisplay-black-webfont.ttf') format('truetype');
	font-weight: normal;
	font-style: normal;
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-		url('./fonts/ProximaNova-Semibold.ttf') format('truetype');
-	font-weight: 600;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-medium-webfont.eot');
+    src: local('creato'), local('creatodisplay-medium-webfont'),
+         url('./fonts/creatodisplay-medium-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-medium-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-medium-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-medium-webfont.ttf') format('truetype');
+    font-weight: 600;
+    font-style: normal;
```
```suggestion
	font-family: 'creato';
	src: url('./fonts/creatodisplay-medium-webfont.eot');
	src: local('creato'),
		local('creatodisplay-medium-webfont'),
		url('./fonts/creatodisplay-medium-webfont.eot?#iefix') format('embedded-opentype'),
		url('./fonts/creatodisplay-medium-webfont.woff') format('woff'),
		url('./fonts/creatodisplay-medium-webfont.woff2') format('woff2'),
		url('./fonts/creatodisplay-medium-webfont.ttf') format('truetype');
	font-weight: 600;
	font-style: normal;
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-		url('./fonts/ProximaNova-Bold.ttf') format('truetype');
-	font-weight: bold;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-regular-webfont.eot');
+    src: local('creato'), local('creatodisplay-regular-webfont'),
+         url('./fonts/creatodisplay-regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-regular-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-regular-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-regular-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
```
```suggestion
	font-family: 'creato';
	src: url('./fonts/creatodisplay-regular-webfont.eot');
	src: local('creato'),
		local('creatodisplay-regular-webfont'),
		url('./fonts/creatodisplay-regular-webfont.eot?#iefix') format('embedded-opentype'),
		url('./fonts/creatodisplay-regular-webfont.woff') format('woff'),
		url('./fonts/creatodisplay-regular-webfont.woff2') format('woff2'),
		url('./fonts/creatodisplay-regular-webfont.ttf') format('truetype');
	font-weight: normal;
	font-style: normal;
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
-		url('./fonts/ProximaNova-Regular.woff') format('woff'),
-		url('./fonts/ProximaNova-Regular.ttf') format('truetype');
-	font-weight: normal;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-black-webfont.eot');
+    src: local('creato'), local('creatodisplay-black-webfont'),
+         url('./fonts/creatodisplay-black-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-black-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-black-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-black-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
 }
```
"Black" fonts should be `font-weight: 900`, not `normal`.

jakekvm01 replied:
Changing it now. Thank you

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,54 @@
+    font-style: normal;
 }
 
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Extrabld.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Extrabld'),
-		url('./fonts/ProximaNova-Extrabld.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Extrabld.woff') format('woff'),
-		url('./fonts/ProximaNova-Extrabld.ttf') format('truetype');
-	font-weight: 800;
-	font-style: normal;
-}
 
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Black.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Black'),
-		url('./fonts/ProximaNova-Black.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Black.woff') format('woff'),
-		url('./fonts/ProximaNova-Black.ttf') format('truetype');
-	font-weight: 900;
-	font-style: normal;
-}
```
Is there a reason that creato extra bold and bold was left out?

jakekvm01 replied:
There's none. Just need a suggestion before adding it. Thank you.

MicahZoltu commented on source/index.html:
```diff
@@ -766,20 +765,88 @@ <h3 class='title'>Pricing</h3>
+					padding-top: 20px;
+					padding-bottom: 20px;
+				}
+                @media screen and (max-width: 680px) {
```
```suggestion
				@media screen and (max-width: 680px) {
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,20 +765,88 @@ <h3 class='title'>Pricing</h3>
+					#addonsbanner > .addonsbanner {
+						grid-column: 1;
+						grid-row: 1;
+            	}
+            </style>
+            <div class='addonsbanner'>
+            <h1> + ADD-ONS </h1>
+            </div>
+        </section>
+        <section id='addons'>
+            <style>
```
```suggestion
				}
			</style>
			<div class='addonsbanner'>
				<h1> + ADD-ONS </h1>
			</div>
		</section>
		<section id='addons'>
			<style>
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,20 +765,88 @@ <h3 class='title'>Pricing</h3>
+					              }
+				}
+			</style>
+                <div class='pricing-specs month'>
+				<div class='amount'>$100</div>
+				<div class='per'><span>Per</span></div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Chat)</div>
+				<button id='addonchat'>Details</button>
+			    </div>
+			    <div class='pricing-specs month'>
+				<div class='amount'>$200</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Non-chat)</div>
+				<button id='addonnonchat'>Details</button>
+                </div>
+	    </section>
```
```suggestion
			<div class='pricing-specs month'>
				<div class='amount'>$100</div>
				<div class='per'><span>Per</span></div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Chat)</div>
				<button id='addonchat'>Details</button>
			</div>
			<div class='pricing-specs month'>
				<div class='amount'>$200</div>
				<div class='per'>Per</div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Non-chat)</div>
				<button id='addonnonchat'>Details</button>
			</div>
		</section>
```

MicahZoltu commented on source/index.html:
```diff
@@ -956,13 +1023,13 @@ <h3 class='title'>Pricing</h3>
 				</div>
-			</div>
-			<div id='question-fee-modal-content' class='closed'>
+			    </div>
+				<div id='question-fee-modal-content' class='closed'>
```
```suggestion
			</div>
			<div id='question-fee-modal-content' class='closed'>
```

MicahZoltu commented on source/index.html:
```diff
@@ -956,13 +1023,13 @@ <h3 class='title'>Pricing</h3>
-					<div class='per'>Per</div>
-					<div class='unit'>Question</div>
-					<div class='freebie'>(First 100 free)</div>
+				<div class='amount'>$3</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Question</div>
+				<div class='freebie'>(First 100 free)</div>
```
```suggestion
					<div class='amount'>$3</div>
					<div class='per'>Per</div>
					<div class='unit'>Question</div>
					<div class='freebie'>(First 100 free)</div>
```

MicahZoltu commented on source/index.html:
```diff
@@ -977,8 +1044,42 @@ <h3 class='title'>Pricing</h3>
-			</div>
-		</div>
+			    </div>
+                <div id='addonchat-content' class='closed'>
+				<div class='pricing-specs'>
+				<div class='amount'>$100</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Chat)</div>
+				</div>
+				<div class='copy'>
+					<p>
+						This is a 100 USD monthly add-on to your bill for every additional chat support platform (e.g Telegram, Intercom).
+					</p>
+					<br/>
+					<p>
+						The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Excess questions will still cost an additional $3 per question/case.
+                                        </p>
+				</div>
+			    </div>
+                <div id='addonnonchat-content' class='closed'>
+				<div class='pricing-specs'>
+				<div class='amount'>$200</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Non-chat)</div>
+				</div>
+				<div class='copy'>
+					<p>
+						This is a 200 USD monthly add-on to your bill for each additional non-chat support platform (e.g Email).
+					</p>
+					<br/>
+					<p>
+						The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Overage per resolved case will be an additional $5.
+                    </p>
+				</div>
+			    </div>                 
+		        </div>
```
```suggestion
			<div id='addonchat-content' class='closed'>
				<div class='pricing-specs'>
					<div class='amount'>$100</div>
					<div class='per'>Per</div>
					<div class='unit'>Month</div>
					<div class='freebie'>(Chat)</div>
				</div>
				<div class='copy'>
					<p>This is a 100 USD monthly add-on to your bill for every additional chat support platform (e.g Telegram, Intercom).</p>
					<br/>
					<p>The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Excess questions will still cost an additional $3 per question/case.</p>
				</div>
			</div>
			<div id='addonnonchat-content' class='closed'>
				<div class='pricing-specs'>
					<div class='amount'>$200</div>
					<div class='per'>Per</div>
					<div class='unit'>Month</div>
					<div class='freebie'>(Non-chat)</div>
				</div>
				<div class='copy'>
					<p>This is a 200 USD monthly add-on to your bill for each additional non-chat support platform (e.g Email).</p>
					<br/>
					<p>The added monthly cost still includes 24×7 quality support service (minus 1 hour lunch breaks in certain cases and catastrophic events). Overage per resolved case will be an additional $5.</p>
				</div>
			</div>
		</div>
```

MicahZoltu reviewed (changes requested):
Mostly style issues, though there are a couple of comments about missing fonts in `fonts.css` that should be commented on or resolved.

jakekvm01 commented:
- Done updating the index.html
- Added thin font in fonts folder.
- Edited and added thin,bold and extra bold in CSS file.

Looking good now http://htmlpreview.github.io/?https://github.com/jakekvm01/landing-page/blob/jakekvm01-patch-1/source/index.html

MicahZoltu commented on source/index.html:
```diff
@@ -766,11 +765,79 @@ <h3 class='title'>Pricing</h3>
+						grid-column: 1;
+						grid-row: 1;
+					
+					              }
```
```suggestion
					}
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,11 +765,78 @@ <h3 class='title'>Pricing</h3>
+					#addonsbanner > .addonsbanner {
+						grid-column: 1;
+						grid-row: 1;
+				}
```
```suggestion
					}
				}
```

MicahZoltu commented on source/fonts.css:
```diff
@@ -1,76 +1,98 @@
+
+
+
+
```
```suggestion
```

MicahZoltu replied:
```suggestion
```

MicahZoltu commented on source/index.html:
```diff
@@ -766,11 +765,79 @@ <h3 class='title'>Pricing</h3>
+				<button id='addonnonchat'>Details</button>
+			</div>
+		</section>
+			<div id='footer-background' class='background'>
```
```suggestion
		<div id='footer-background' class='background'>
```
